.github/pull_request_template.md

```diff
@@ -1,15 +1,15 @@
-<!-- Which issue(s) does this pull request fix or resolve? -->
+<!-- Which issue(s) does this pull request fix or resolve? If there aren't any, please submit one first unless this is a hotfix or minor string update. -->
 
 Resolves #
 
 ### Changes
 
-<!-- Please describe the changes you've made. -->
+<!-- Please describe the changes you've made. Add any screenshots or videos here if applicable. -->
 
 ### Reason for changes
 
 <!-- Why should these changes be made? -->
 
 ### Tests
 
-<!-- If applicable. Have you tested this pull request? If so, how? -->
+<!-- Please test your changes in a Chromium-based browser and add any known issues or other testing notes here. Bigger changes should also be tested on Firefox. -->
```

Comment:
Maybe it would be better if we worded this to include PR authors that use Firefox as their main browser. The requirement is to test it in 1 browser, not to test it in a Chromium-based one 🤔